src/common/variables.js

```diff
@@ -0,0 +1,13 @@
+
+
+
+
+export const fotFunction = (count, array, component) => {
+	const newArray = [...array];
+
+	for (let i = 0; i < count; i++) {
```

Comment:
I don`t understand what this function is doing by its name, it`s confusing.

BTW - if you only want to fill an array with the same component you can do it like this : 

```js
(count, array, component) => [...array, ...Array(count).fill(component)];
```